ddtrace/tracer/tracer_test.go

```diff
@@ -1038,6 +1039,22 @@ func BenchmarkTracerAddSpans(b *testing.B) {
 	}
 }
 
+func BenchmarkStartSpan(b *testing.B) {
```

Comment:
Should have a benchmark for measuring the adding of metrics and metadata?

Reply:
We do. There is a small difference, but at least everything happens in the right place now (on a need-to basis):

```
benchmark                   old ns/op     new ns/op     delta
benchmark                   old ns/op     new ns/op     delta
BenchmarkSetTagMetric-8     67.3          65.6          -2.53%
BenchmarkSetTagString-8     60.4          62.5          +3.48%
BenchmarkSetTagField-8      37.4          39.4          +5.35%

```